java/cli/src/main/java/org/ray/cli/CommandStart.java

```diff
@@ -0,0 +1,24 @@
+package org.ray.cli;
+
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.Parameters;
+
+/**
+ * Arguments for command start
+ */
+@Parameters(separators = "= ", commandDescription = "start ray daemons")
+public class CommandStart {
+
+  @Parameter(names = "--head", description = "start the head node")
+  public boolean head;
+
+  @Parameter(names = "--work", description = "start the work node including local scheduler, plasma and worker")
```

Comment:
Should this be `worker`?

Reply:
Just wondering if it would confuse with worker process if we use worker here?